src/main/resources/edu/cornell/kfs/fp/document/datadictionary/RecurringDisbursementVoucherDocument.xml

```diff
@@ -130,6 +131,31 @@
 		</property>
 	</bean>
 
+	<bean id="RecurringDisbursementVoucherDocument-validations" parent="RecurringDisbursementVoucherDocument-validations-parentBean" />
+	<bean id="RecurringDisbursementVoucherDocument-validations-parentBean" abstract="true" 
+		parent="DisbursementVoucherDocument-validations-parentBean" class="org.springframework.beans.factory.config.MapFactoryBean">
+		<property name="sourceMap">
+			<map key-type="java.lang.Class">
```

Comment:
Is it possible to use the merge="true" option here, so that you don't have to copy all of the map entries from the parent bean?

Reply:
yes, I should be able to use the merge option, good idea